commons/src/test/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitTasksTest.scala

```diff
@@ -83,40 +83,41 @@ class PurePursuitTasksTest extends FunSuite {
   }
 
   val unlimitedTurnOutput = Percent(Double.MaxValue)
-  test("If target is initally within tolerance, stop immediately") {
-    var lastAppliedSignal = zeroSignal
-    val drivetrainComp = new TestDrivetrainComponent(lastAppliedSignal = _)
-
-    implicit val hardware = new UnicycleHardware {
-      override val forwardVelocity: Stream[Velocity] = Stream.periodic(period)(MetersPerSecond(0))
-      override val turnVelocity: Stream[AngularVelocity] = Stream.periodic(period)(DegreesPerSecond(0))
-
-      override val forwardPosition: Stream[Length] = Stream.periodic(period)(Feet(0))
-      override val turnPosition: Stream[Angle] = Stream.periodic(period)(Degrees(0))
-    }
-
-    val target = new Point(Feet(0), Feet(0.5))
-    val task = new drive.unicycleTasks.FollowWayPoints(
-      Seq(Point.origin, target),
-      Feet(1),
-      unlimitedTurnOutput
-    )(drivetrainComp)
-
-    task.init()
-
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-
-    assert(!task.isRunning)
-  }
+//  test("If target is initally within tolerance, stop immediately") {
```

Comment:
@Paxelord @RinHa post-merge: why was this test disabled? does the robot drive when it is within the tolerance?